connector-pagerduty/src/main/java/com/redhat/cloud/notifications/connector/pagerduty/PagerDutyTransformer.java

```diff
@@ -80,14 +92,38 @@ public void process(Exchange exchange) {
         exchange.getIn().setBody(message.encode());
     }
 
+    /** Validates that the inputs for the required Alert Event fields are present */
+    private void validatePayload(final JsonObject cloudEventPayload) {
+        String summary = cloudEventPayload.getString(EVENT_TYPE);
+        if (summary == null || summary.isEmpty()) {
+            throw new IllegalArgumentException("Event type must be specified for PagerDuty payload summary");
+        }
+
+        String source = cloudEventPayload.getString(APPLICATION);
+        if (source == null || source.isEmpty()) {
+            throw new IllegalArgumentException("Application must be specified for PagerDuty payload source");
+        }
```

Comment:
Both the Event Type and the Application are always present on the Cloud Event, so I think we can spare these checks

Reply:
yeah this is an outdated check, will remove